Update settingtypes (#13428)
minetest/minetest: builtin/settingtypes.txt

[*General]

#    Smooths rotation of camera, also called look or mouse smoothing. 0 to disable.
camera_smoothing (Camera smoothing) float 0.0 0.0 0.99

> #    Smooths rotation of camera when in cinematic mode, 0 to disable. Enter cinematic mod by using the key set in Change Keys.

Author: ```suggestion
#    Smooths rotation of camera when in cinematic mode, 0 to disable. Enter cinematic mode by using the key set in Change Keys.
```

cinematic_camera_smoothing (Camera smoothing in cinematic mode) float 0.7 0.0 0.99

> #    If enabled, you can place blocks at the position (feet + eye level) where you stand.

Reviewer: nodes*

Author: ```suggestion
#    If enabled, you can place nodes at the position (feet + eye level) where you stand.
```

#    A restart is required after changing this option.
fsaa (FSAA) enum 0 0,1,2,4,8,16

[**Occlusion Culling]

#    Type of occlusion_culler

> #    "loops" is the legacy algorithm with nested loops and O(N^3) complexity

Reviewer: ```suggestion
#    "loops" is the legacy algorithm with nested loops and O(n³) complexity
```

also shouldn't this indicate that the option is to be used in case of performance problems only?

Author: that's correct

#    type: float min: -60 max: 60
shadow_sky_body_orbit_tilt (Sky Body Orbit Tilt) float 0.0 -60.0 60.0

[**Post Processing]

#    Enables Hable's 'Uncharted 2' filmic tone mapping.
#    Simulates the tone curve of photographic film and how this approximates the
#    appearance of high dynamic range images. Mid-range contrast is slightly
#    enhanced, highlights and shadows are gradually compressed.
tone_mapping (Filmic tone mapping) bool false

#    Enable automatic exposure correction
#    When enabled, the post-processing engine will
#    automatically adjust to the brightness of the scene,
#    simulating the behavior of human eye.
enable_auto_exposure (Enable Automatic Exposure) bool false

#    Set the exposure compensation in EV units.
#    Value of 0.0 (default) means no exposure compensation.
#    Range: from -1 to 1.0

> exposure_compensation	(Exposure compensation) float 0.0 -1.0 1.0

Reviewer: ```suggestion
exposure_compensation (Exposure compensation) float 0.0 -1.0 1.0
```

#    Changing this setting requires a restart.

> enable_sound (Sound) bool true

Reviewer 2: Should be moved to `[*Audio]`. (Ironically, this is the only non-tmp setting of the three.)

Author: No, this is intentionally not visible in the GUI as it's redundant with mute sound

Reviewer 2: Oh, so the ingame audio menu shows exactly this section? Didn't know.

Reviewer 2: Ah, so `[*Temporary Settings]` is actually `[*Hidden Settings]`. I see.

#    Unix timestamp (integer) of when the client last checked for an update
#    Set this value to "disabled" to never check for updates.
update_last_checked (Last update check) string

#    Version number which was last seen during an update check.
#
#    Representation: MMMIIIPPP, where M=Major, I=Minor, P=Patch
#    Ex: 5.5.0 is 005005000
update_last_known (Last known version update) int 0

#    This can be bound to a key to toggle camera smoothing when looking around.
#    Useful for recording videos

> cinematic (Cinematic mode) bool false

Reviewer 2: Would suit a little better directly below `continuous_forward`, as it's a tmp setting that one changes in-game via a key.

Author: It's already in that section, I haven't bothered sorting them in any particular way
